nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/test/java/org/apache/nifi/util/db/TestJdbcTypesH2.java

```diff
@@ -75,9 +71,19 @@ public static void setup() {
 
     String dropTable = "drop table users";
 
+    String dbPath;
+
+    @BeforeEach
+    public void beforeEach() throws IOException {
+        dbPath = Files.createTempDirectory(String.valueOf(System.currentTimeMillis()))
+                .resolve("db")
+                .toFile()
+                .getAbsolutePath();
```

Comment:
This approaches creates a temporary directory, but does not appear to delete the contents after test completion.  There should be a corresponding `@AfterEach` method to remove the directory contents. See `FileUtils.deleteFile(File, boolean)` in `nifi-utils` for recursive deletion of a directory and contents.